planner/core/physical_plan_test.go

```diff
@@ -590,7 +544,7 @@ func (s *testPlanSuite) TestINMJHint(c *C) {
 		dom.Close()
 		store.Close()
 	}()
-	tk := testkit.NewTestKit(c, store)
+	tk := testkitToRemove.NewTestKit(c, store)
```

Comment:
all the changes below in this file are just renaming. Will remove testkitToRemove package in following prs.